frontend/pages/gameboard.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="../styles/gameboard.css" rel="stylesheet"/>
+</head>
+<body>
+    <h1 {text-align: centre;} style="Color:White;"> 🆃🅴🆃🆁🅸🆂
```

Comment:
We can get rid of the curly brackets, and the text-align: centre here. We already do this in the css file!